components/form/form-error-summary.js

```diff
@@ -54,6 +58,7 @@ class FormErrorSummary extends LocalizeCoreElement(LitElement) {
 		super();
 		this.errors = [];
 		this._expanded = true;
+		this._hasBottomMargin = false;
```

Comment:
Should also set `this._hasErrors = false;`. (I realize I forgot that in my PR as well! 😅 )